test/patches/convertInlinePoliciesToManaged.test.ts

```diff
@@ -78,6 +78,37 @@ describe('Updating Resource Types', () => {
     );
   });
 
+  test('Function dependencies on policy are maintained', () => {
+    const fn = new Function(stack, 'TestLambda', {
+      code: Code.fromInline(`def handler(event, context)\n    print(event)`),
+      handler: 'index.handler',
+      runtime: Runtime.PYTHON_3_11,
+    });
+    fn.addToRolePolicy(
+      new iam.PolicyStatement({
+        actions: ['s3:*'],
+        resources: ['*'],
+      })
+    );
+
+    Aspects.of(app).add(new ConvertInlinePoliciesToManaged());
+    app.synth();
+    const template = Template.fromStack(stack);
+    console.log((template as any).template.Resources);
+    console.log(
+      (template as any).template.Resources
+        .TestLambdaServiceRoleDefaultPolicy0F2D5E78
+    );
```

Comment:
Can you remove the console logs in the test?

Reply:
Ah yea, good call. I'll update